data/transport missions.txt

```diff
@@ -1868,6 +1872,309 @@ mission "Terraforming 5"
+			`	Before you are able to leave, Amy stops you. "Could you meet me in the spaceport bar, Captain? I have some good news to share with you."`
+
+
+
+mission "Terraforming 7"
+	name "Terraforming Research"
+	description "Bring Amy and Nolan to <destination> where they will research deeper into Amy's terraforming methods."
+	destination Valhalla
+	passengers 2
+	blocked "You need <capacity> in order to take on the next mission. Return here when you have the required space free."
+	to offer
+		has "Terraforming 6: done"
+	
+	on offer
+		conversation
+			`You find Amy in deep discussion with someone about her academy project, "Affordable Terraforming through Equilibrium Mapping." She takes a sip of her drink and invites you to sit down with them.`
+			`	"Hello, Captain! This is Nolan. He works for the Deep, and he contacted me after hearing about how we started terraforming Rand."`
+			`	Nolan outstretches his hand to you. "Nice to meet you, Captain."`
+			choice
+				`	"Nice to meet you too."`
+					goto next
+				`	"I take it you're interested in Amy's ideas."`
+			
+			`	"Absolutely. It's an extraordinary concept, and no offense to you Amy, but it's spectacular that someone of such little experience would imagine such a method of terraforming."`
+			
+			label next
+			`	"Nolan wants to help pursue research of my terraforming techniques. Would you be able to give us a lift to <destination>? I'll pay you <payment>."`
+			choice
+				`	"Sure, I'd be glad to take you there."`
+					accept
+				`	"Sorry, I have other places to be."`
+					decline
+	
+	on complete
+		payment 50000
+		event "terraforming research" 120 180
+		dialog `The whole journey to <planet>, Amy and Nolan discussed the multiple planets within the Dirt Belt that would be perfect candidates for terraforming. "Nolan and I will spend the next few months here researching my theories on terraforming," Amy says as she steps off of your ship. "I'll make sure to contact once we're ready to try terraforming another planet." She hands you <payment>, then follows Nolan out of the spaceport.`
+
+
+
+event "terraforming research"
+
+mission "Terraforming 8"
+	landing
+	name "Terraforming Research"
+	description "Meet with Amy and Nolan on <destination> to talk about experimentally terraforming another planet."
+	destination Valhalla
+	to offer
+		has "event: terraforming research"
+	
+	on offer
+		dialog `You receive a message from Amy: "Hello, Captain. I just wanted to tell you that Nolan and I have recently published our work on 'Affordable Terraforming through Equilibrium Mapping', and we've received a grant from the Deep for it! We're going to begin experiment with terraforming on a different world. Don't worry, the Republic has approved our experiment this time, so we won't get dragged off for any more cases of an 'unlawful use of an asteroid.' If you're interested in helping, pick us up on <destination>."`
```

Comment:
Research papers tend to not have the same name as any previous research (published or not), a paper typically gets increasingly long titles as it goes through these big size changes. So something like 'Using Equilibrium Mapping for Wide Scale Terraforming' or if that's too long 'Reduce Terraforming Expense Through Equilibrium Mapping'. Either way I find it odd to be reintroduced to the same title for a different paper.

Also, the full stop on the second to last sentence seems to have slipped inside the quotation marks.

Reply:
`Equilibrium Mapping', and` should be `Equilibrium Mapping,' and` because this is written in American English.

In the case of that second-to-last period, that actually conforms to American English rules.